File: csharp/get_campaign_statut.cs
//replace with your own values

var client = new RestClient("http://localhost:3333/v1/campaigns/1/status");
client.Timeout = -1;
var request = new RestRequest(Method.GET);
request.AddHeader("Authorization", "Bearer dc408be00542eed70f4e7d1336d9a38735e5fa74");
var body = @"";
request.AddParameter("text/plain", body, ParameterType.RequestBody);
IRestResponse response = client.Execute(request);
Console.WriteLine(response.Content);

File: js/get_campaign_statut.js
//replace with your own values
//AXIOS VERSION
//install axios first => npm install axios

var axios = require("axios");

var config = {
  method: "get",
  url: "http://localhost:3333/v1/campaigns/1/status",
  headers: {
    Authorization: "Bearer dc408be00542eed70f4e7d1336d9a38735e5fa74",
    "Content-Type": "application/json",
  },
};

axios(config)
  .then(function (response) {
    console.log(JSON.stringify(response.data));
  })
  .catch(function (error) {
    console.log(error);
  });
